sys-kernel/installkernel/installkernel-39.ebuild

```diff
@@ -17,11 +17,12 @@ S="${WORKDIR}/${PN}-gentoo-${PV}"
 LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~x86-linux"
-IUSE="dracut efistub grub refind systemd systemd-boot uki ukify"
+IUSE="dracut efistub grub refind systemd systemd-boot ugrd uki ukify"
```

Comment:
If we add a new flag, then we must add a description for that flag to installkernel's `metadata.xml` (see that file for an example on how to do it). This is what the CI is complaining about: `UnknownUseFlags`

Reply:
I think this is fixed now